can_tools/scrapers/official/SD/sd_vaccines.py

```diff
@@ -0,0 +1,279 @@
+import json
+
+from typing import Any
+
+import pandas as pd
+import us
+
+from can_tools.scrapers import CMU, variables
+from can_tools.scrapers.util import flatten_dict
+from can_tools.scrapers.official.base import MicrosoftBIDashboard
+
+"""
+    Notes:
+    J&J shot included in "one shot" data, but NOT in two shot data
+"""
+
+
+class SDVaccineCounty(MicrosoftBIDashboard):
+
+    has_location = False
+    location_type = "county"
+    state_fips = int(us.states.lookup("South Dakota").fips)
+
+    source = "https://doh.sd.gov/COVID/Dashboard.aspx"
+    source_name = "South Dakota Department of Health"
+    powerbi_url = "https://wabi-us-gov-iowa-api.analysis.usgovcloudapi.net"
+
+    variables = {
+        "at_least_one_dose": variables.INITIATING_VACCINATIONS_ALL,
+        "total_doses": variables.TOTAL_DOSES_ADMINISTERED_ALL,
+    }
+
+    def construct_body(self, resource_key, ds_id, model_id, report_id):
+        "Build body request"
+        body = {}
+
+        # Set version
+        body["version"] = "1.0.0"
+        body["cancelQueries"] = []
+        body["modelId"] = model_id
+
+        # used the orignal query (from the network call) because without the "WHERE" clause the # of vaccines initiated were not returned
+        body["queries"] = [
+            {
+                "Query": {
+                    "Commands": [
+                        {
+                            "SemanticQueryDataShapeCommand": {
+                                "Binding": {
+                                    "DataReduction": {
+                                        "DataVolume": 3,
+                                        "Primary": {"Window": {"Count": 100}},
+                                        "Secondary": {"Top": {"Count": 100}},
```

Comment:
Do you know if this is limiting the number of rows returned?

Reply:
that does look like it might. This query returns all of the data completely, it was a query I made using the `MicrosoftBIDashboard` `_construct_...()` functions that was not returning all the data. 

When I increased those values (to 1000) the response had all the same data, but also returned a warning: 
`[{'Code': 'SpecifiedReductionAlgorithmsExceedsMaxIntersections', 'Severity': 'Warning', 'Message': 'Specified reduction algorithms exceed the maximum allowed number of intersections (1000000). The counts will be ignored.'}]`

Reply:
Hmm...ok. Do the other PowerBI scrapers include this "filter" as well?

Reply:
I've found similar filters in some queries of other BI dashboards, but it is not universal. Below is an example of a query from the Minnesota dashboard. I haven't found a pattern as to why it's included in some instances and not others--but I can keep searching.

![image](https://user-images.githubusercontent.com/55333380/113430690-947de700-93a8-11eb-9ae0-ff73ca59f090.png)

Reply:
If you haven't found anything contradictory, and the totals are matching the dashboard, then I'll approve this

Reply:
@tlyon3 the values match, so this should be good to go. (only scrapes people initiating vaccine and total doses administered)